package.json

```diff
@@ -17,7 +17,8 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "jest --config ./test/jest-e2e.json",
```

Comment:
at the top of package.json, under license let's add:
![image](https://github.com/SeattleColleges/belindas-closet-nestjs/assets/64063020/95155387-fcf2-43d5-8a79-8c6104a8cdea)
 > This will warn devs if they are not using node 18.